missioncontrol/commands/acquirelicense.go

```diff
@@ -0,0 +1,60 @@
+package commands
+
+import (
+	"encoding/json"
+	"errors"
+	"github.com/jfrog/jfrog-cli-go/missioncontrol/utils"
+	"github.com/jfrog/jfrog-cli-go/utils/cliutils"
+	"github.com/jfrog/jfrog-cli-go/utils/config"
+	"github.com/jfrog/jfrog-client-go/httpclient"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"net/http"
+)
+
+func AcquireLicense(bucketId string, name string, mcDetails *config.MissionControlDetails) error {
+	postContent := AcquireLicenseRequestContent{
+		Name:         name,
+		LicenseCount: 1,
+	}
+	requestContent, err := json.Marshal(postContent)
+	if err != nil {
+		return errorutils.CheckError(errors.New("Failed to marshal json. " + cliutils.GetDocumentationMessage()))
```

Comment:
I suggest just returning the err (with CheckError)  and let cli.go handle the "see documentation" message. This is the approach we took in the Artifactory commands.
This suggestion applies for the other MC commands as well.